src/kex_sidh_cln16/kex_sidh_cln16.c

```diff
@@ -126,34 +127,33 @@ int OQS_KEX_sidh_cln16_alice_0(OQS_KEX *k, void **alice_priv, uint8_t **alice_ms
 
 err:
 	ret = 0;
-	if (alice_msg && *alice_msg) {
```

Comment:
All contemporary implementations of `free()` can be called with a NULL pointer. The majority of code in liboqs does not check to see if the pointer is NULL before calling `free()`.

Reply:
Good point, thanks.